packages/fxa-payments-server/src/components/Header/en.ftl

```diff
@@ -1,3 +1,4 @@
 ## Component - Header
 
 brand-name-firefox-logo = { -brand-name-firefox } logo
+brand-name-mozilla-logo = { -brand-mozilla } logo
```

Comment:
Based on Storybook it appears terms from `shared/branding.ftl` are working with payments as well.